src/Resources/config/doctrine-phpcr/Route.phpcr.xml

```diff
@@ -13,6 +13,7 @@
         <nodename name="name"/>
         <parent-document name="parent"/>
         <children name="children"/>
+        <child-class name="Symfony\Cmf\Component\Routing\RouteObjectInterface"/>
```

Comment:
We have a couple of options here:

 * Restrict to this interface
 * Restrict to `Symfony\Component\Routing\Route`
 * Restrict to the `Route` and `RedirectRoute` models of this bundle

I choose the first option, as it seems to be the most flexible without completely allowing anything.